distribution/lib/Standard/Snowflake/0.0.0-dev/src/Internal/Snowflake_Dialect.enso

```diff
@@ -584,6 +584,10 @@ starts_with = Base_Generator.lift_binary_sql_function "STARTS_WITH" "STARTSWITH"
 ## PRIVATE
 ends_with = Base_Generator.lift_binary_sql_function "ENDS_WITH" "ENDSWITH"
 
+## PRIVATE
+regex_match = Base_Generator.lift_binary_op "REGEX_MATCH" str-> pattern->
+    str ++ " REGEXP " ++ pattern
```

Comment:
I'm wondering if it may be safer to wrap in parentheses to avoid risking some operator precedence issues.

Reply:
Done